cmd/neofs-adm/internal/modules/morph/initialize.go

```diff
@@ -381,13 +381,20 @@ loop:
 	return retErr
 }
 
-// sendCommitteeTx creates transaction from script and sends it to RPC.
+// sendCommitteeTx creates transaction from script, signs it by committee nodes and sends it to RPC.
 // If tryGroup is false, global scope is used for the signer (useful when
 // working with native contracts).
 func (c *initializeContext) sendCommitteeTx(script []byte, tryGroup bool) error {
 	return c.sendMultiTx(script, tryGroup, false)
 }
 
+// sendConsensusTx creates transaction from script, signs it by alphabet nodes and sends it to RPC.
+// Not that because this is used only after the contracts were initialized and deployed,
```

Comment:
```suggestion
// Note that because this is used only after the contracts were initialized and deployed,
```

or i did not get the idea of the comment

Reply:
Fixed